exchanges/kucoin/kucoin_test.go

```diff
@@ -1156,11 +1156,20 @@ func TestGetFuturesContract(t *testing.T) {
 	}
 }
 
-func TestGetFuturesRealTimeTicker(t *testing.T) {
+func TestGetFuturesTicker(t *testing.T) {
 	t.Parallel()
-	_, err := ku.GetFuturesRealTimeTicker(context.Background(), "XBTUSDTM")
-	if err != nil {
-		t.Error("GetFuturesRealTimeTicker() error", err)
+	tick, err := ku.GetFuturesTicker(context.Background(), "XBTUSDTM")
+	if assert.NoError(t, err, "GetFuturesTicker should not error") {
+		assert.Equal(t, "XBTUSDTM", tick.Symbol, "Symbol should be correct")
```

Comment:
Looks like `assert.Positive(t, tick.Sequence, "Sequence should be positive")` was purged in 6488b08fb0c03c148eaaa3689dc12861c712bc94

Reply:
Fixed. Thanks.

Suspect that was `rerere`